### DISCUSSION TITLE
Flux HelmRelease extracted dependencies for OCI repositories are documented as type "helm" instead of "docker"

### DISCUSSION THREAD

**Discussion by gberche-orange** [renovatebot/renovate#24185] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

The documentation states :

https://github.com/renovatebot/renovate/blob/e066924971bb0dc4ebfa0c95c919969c9eba7064/lib/modules/manager/flux/readme.md?plain=1#L12

However, with OCI HelmRepo support contributed by @samip5 in https://github.com/renovatebot/renovate/pull/22291 I see that the extracted dependency is of type "docker" (see below)

The unit test also seems to instead expect a type docker for extraced dependency
https://github.com/renovatebot/renovate/blob/3c3f2464481bc2ab0f1cf9c292478ea99a56b695/lib/modules/manager/flux/extract.spec.ts#L731-L751

I'm opening this discussion to confirm the expected datasource before fixing the doc if needed

### Relevant debug logs

<details><summary>Logs</summary>

```
{
        "packageFile": "micro-depls/00-core-connectivity-k8s/k8s-config/manifests/10-harbor-registry-main/harbor-helm-release.yaml",
        "deps": [
          {
            "depName": "harbor",
            "currentValue": "15.2.5",
            "datasource": "docker",
            "packageName": "registry-1.docker.io/bitnamicharts/harbor",
            "updates": [
              {
                "bucket": "non-major",
                "newVersion": "15.3.0",
                "newValue": "15.3.0",
                "newMajor": 15,
                "newMinor": 3,
                "updateType": "minor",
                "branchName": "renovate/harbor-15.x"
              },
              {
                "bucket": "major",
                "newVersion": "17.1.0",
                "newValue": "17.1.0",
                "newMajor": 17,
                "newMinor": 1,
                "updateType": "major",
                "branchName": "renovate/harbor-17.x"
              }
            ],
            "versioning": "docker",
            "warnings": [],
            "registryUrl": "https://registry-1.docker.io",
            "currentVersion": "15.2.5",
            "isSingleVersion": true,
            "fixedVersion": "15.2.5"
          }
        ]
      },
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by samip5** [comment]
Tehnically OCI repos are Helm, but I chose to use Docker dep so the upgrading of those works properly. I'm not sure if it would if it used helm source type.

**Reply by viceice** [reply]
OCI charts are stored in docker registry, so renovate uses the docker datasource which is adding the docker tag stuff to the commit message by default.

**Reply by gberche-orange** [reply]
Thanks to both of you. Makes sense to me to just update the doc to reflect current behavior., I'm submitting a small doc fix PR referencing this discussion.

**Reply by gberche-orange** [reply]
https://github.com/renovatebot/renovate/pull/24197
